client/components/TestRun/index.jsx

```diff
@@ -192,14 +209,97 @@ class TestRun extends Component {
+                            {this.state.showTestNavigator ? (
+                                <>
+                                    <h3>Test Navigator</h3>
+                                    <button
+                                        onClick={this.toggleTestNavigator}
+                                        className="test-navigator-toggle hide"
+                                    >
+                                        Hide
+                                    </button>
+                                    <ol className="test-navigator-list">
+                                        {run.tests.map((t, i) => {
+                                            let resultClassName = 'not-started';
+                                            const testersResult =
+                                                t.results &&
+                                                t.results[openAsUser || userId];
+                                            if (testersResult) {
+                                                if (
+                                                    testersResult.status ==
+                                                    'incomplete'
+                                                ) {
+                                                    resultClassName =
+                                                        'in-progress';
+                                                } else if (
+                                                    testersResult.status ==
+                                                    'skipped'
+                                                ) {
+                                                    resultClassName = 'skipped';
+                                                } else if (
+                                                    checkForConflict(t.results)
+                                                        .length
+                                                ) {
+                                                    resultClassName =
+                                                        'conflicts';
+                                                } else if (
+                                                    testersResult.status ===
+                                                    'complete'
+                                                ) {
+                                                    resultClassName =
+                                                        'complete';
+                                                }
+                                            } else if (
+                                                this.state.currentTestIndex -
+                                                    1 >
+                                                i
+                                            ) {
+                                                resultClassName = 'skipped';
```

Comment:
@isaacdurazo - I'm not sure how we should define what "skipped" means. I wonder if it should show up separately at all. 

I took a stab at it but we should discuss.

Reply:
Remove skipped.